topic-controller/src/main/java/io/strimzi/controller/topic/Config.java

```diff
@@ -196,11 +196,11 @@ public Config(Map<String, String> map) {
     }
 
     public <T> T get(Value<T> value, T defaultValue) {
-        return (T)this.map.getOrDefault(value.key, defaultValue);
+        return (T) this.map.getOrDefault(value.key, defaultValue);
```

Comment:
I never understood how the space helps here (just saying).

Reply:
Me neither!